dtfs-central-api/src/services/repositories/durable-functions-repo.ts

```diff
@@ -0,0 +1,15 @@
+import { Collection, DeleteResult } from 'mongodb';
+import db from '../../drivers/db-client';
+import { DB_COLLECTIONS } from '../../constants';
+
+const getDurableFunctionsLogCollection = async (): Promise<Collection> => db.getCollection(DB_COLLECTIONS.DURABLE_FUNCTIONS_LOG);
+
+export const deleteAllDurableFunctionLogs = async (): Promise<DeleteResult> => {
+  const durableFunctionLogsCollection = await getDurableFunctionsLogCollection();
+  return await durableFunctionLogsCollection.deleteMany({});
```

Comment:
Think the await is unnecessary & the type `Promise<DeleteResult>` will be inferred so doesn't need to be stated. Happy to leave if you think improves readability

Reply:
same for line 14 too

Reply:
Very true